telemetry/definitions/clouddebugDefinitions.json

```diff
@@ -0,0 +1,87 @@
+{
+    "types": [
+        {
+            "name": "cloudDebugPlatform",
```

Comment:
seems overwrought to have separate types for `cloudDebugPlatform`, `runtime`, `schemaLanguage`. Just like we don't have a separate type for `int` ranges, we could have 1 type that covers the general idea of a "platform".

This would make filtering easier.

Reply:
We have weird types currently for cloudDebugPlatform and schema language I don't think they really overlap. The type names here typically come from somewhere else so I'm hesitant to change them

Reply:
That's unfortunate, but if this is a legacy constraint then it is what it is :)